flag.go

```diff
@@ -881,7 +887,29 @@ func (f *FlagSet) parseArgs(args []string) (err error) {
 func (f *FlagSet) Parse(arguments []string) error {
 	f.parsed = true
 	f.args = make([]string, 0, len(arguments))
-	err := f.parseArgs(arguments)
+	err := f.parseArgs(arguments, nil)
```

Comment:
parseShortArg() need to handle "-flag1flag2",  I guess that it need to parse the flag again then to do `f.setFlag`.